addons/website/static/src/js/editor/wysiwyg_multizone.js

```diff
@@ -200,6 +192,18 @@ var WysiwygMultizone = Wysiwyg.extend({
             self.$('[data-oe-readonly]').addClass('o_not_editable').attr('contenteditable', false);
             self.$('.oe_structure').attr('contenteditable', false).addClass('o_fake_not_editable');
             self.$('[data-oe-field][data-oe-type="image"]').attr('contenteditable', false).addClass('o_fake_not_editable');
+
+            // Init destroy
+            var $dropdownToggle = $('.dropdown-toggle');
+            $dropdownToggle.removeAttr('data-toggle').dropdown('dispose');
```

Comment:
So you destroy all the `dropdown-toggle`, including the ones in the editor top bar and stuff. Please search in the right element, not in the whole DOM. What is the `this.$el` here? Is not this the editable zone? And maybe you just want to target only the mega menu dropdown toggles, no?